docs/contributing.rst

```diff
@@ -132,6 +132,11 @@ another ``psql`` commands::
 
 For additional ``psql`` commands see ``man psql``.
 
+To run cron job in vagrant container run these commands::
+
+    $ workon anitya
+    $ ANITYA_WEB_CONFIG=/home/vagrant/anitya.toml python files/anitya_cron.py
```

Comment:
You could make it so that the ```workon anitya``` adds ```ANITYA_WEB_CONFIG=/home/vagrant/anitya.toml``` to the environment automatically.

Reply:
I added it to `bashrc` for vagrant guest.